src/constants/StyleConstants.ts

```diff
@@ -1,10 +1,12 @@
 const basicGrey = "rgb(236,236,236)";
 const trueWhite = "white";
+const niceBlue = "rgba(45, 65, 123, 1)";
 
 export const TAG_CONNECTOR = " • ";
 export const STYLES = {
   COLORS: {
     ODD_CELL: trueWhite,
     EVEN_CELL: basicGrey,
+    ACTIVE_UNDERLINE: niceBlue,
```

Comment:
Could these colors be moved to and merged with src/styles/colors.ts?

Reply:
Yep, importing them from styles/colors.ts.  I don't know how to import the colors by themselves from cdp-design - the package seems designed to wrap them up into css.